app/v2/notifications/post_notifications.py

```diff
@@ -267,12 +269,18 @@ def process_letter_notification(*, letter_data, api_key, template, reply_to_text
                                                         template=template,
                                                         reply_to_text=reply_to_text)
 
+    postcode = letter_data['personalisation']['postcode']
+    if not is_a_real_uk_postcode(postcode):
+        raise ValidationError(message='A real UK postcode is required.')
```

Comment:
Has this message been reviewed by @karlchillmaid or @quis ?

Reply:
not yet, I will ask Chris for a review.

Reply:
I’d suggest
```suggestion
        raise ValidationError(message='postcode must be a real UK postcode`')
```

This uses the same terminology as we’re using on the frontend:
![image](https://user-images.githubusercontent.com/355079/76965841-ec23ee80-691c-11ea-9787-52c1e6e1c534.png)

And the tone of the message matches how we communicate other validation-type errors:
![image](https://user-images.githubusercontent.com/355079/76965930-15dd1580-691d-11ea-8151-46d2448badff.png)

`postcode` should be lowercase because it’s referring to the name of the field in the JSON.